src/lib/components/AimeVideoPlayer/AimeVideoPlayer.js

```diff
@@ -0,0 +1,96 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import ReactPlayer from "react-player";
+import {
+  componentPropTypes,
+  defaultComponentPropTypes,
+} from "../../utils/componentPropTypes";
+import styles from "./AimeVideoPlayer.module.scss";
+
+export default class AimeVideoPlayer extends PureComponent {
```

Comment:
about the naming, I think just `VideoPlayer` is good enough, otherwise we would name the `Button` `AimeButton` and the `Input` `AimeInput` aaand … you get the idea. Unless we would have another VideoPlayer that is "not AIME"?

Reply:
sorry … I now get that you probably just did it to differentiate it from the existing `VideoPlayer` component. I'm guessing that we'd replace that one once this PR is good, but until then, the naming is ok :) .

Reply:
I can always improve my naming conventions! #naminConvs